src/land/ascat_ssm2ioda.py

```diff
@@ -118,11 +123,14 @@ def _read(self):
         self.outdata[('topographyComplexity', 'MetaData')] = tflg
 
         for iodavar in ['soilMoistureNormalized']:
-            self.outdata[self.varDict[iodavar]['valKey']] = vals
-            self.outdata[self.varDict[iodavar]['errKey']] = errs
+            # ioda conventions expect unit 1 not percent (2022/03/09)
+            self.outdata[self.varDict[iodavar]['valKey']] = vals/100.
+            # ioda conventions expect unit 1 not percent (2022/03/09)
+            self.outdata[self.varDict[iodavar]['errKey']] = errs/100.
             self.outdata[self.varDict[iodavar]['qcKey']] = qflg
-        DimDict['nlocs'] = len(self.outdata[('datetime', 'MetaData')])
-        AttrData['nlocs'] = np.int32(DimDict['nlocs'])
+        DimDict['Location'] = len(self.outdata[('datetime', 'MetaData')])
+        # The Location global attribute is deprecated (2022/03/09)
+#       AttrData['Location'] = np.int32(DimDict['Location'])
```

Comment:
Same comment, remove two lines

Reply:
This has been resolved by @gthompsnJCSDA. Please, see the commits below.